docs/maintainer/infrastructure.md

```diff
@@ -695,6 +688,19 @@ In this case, you can use our [admin-requests repo](https://github.com/conda-for
 to add a glob pattern that matches the new package name pattern. We use the Python `fnmatch` module syntax.
 Output packages that match these patterns will be automatically registered for your feedstock.
 
+### Package validation failed for an existing output name
+
+We attempt to report errors in this process to users via comments on commits/issues in the feedstocks.
+Sometimes these reports fail. If you think you are having trouble with uploads, make sure to check/try
+the following things:
+
+- Ensure that `conda_forge_output_validation: true` is set in your `conda-forge.yml`.
+- Retry the package build and upload by pushing an empty commit to the feedstock.
+- Rerender the feedstock in a PR from a fork of the feedstock and merge.
+- Request a feedstock token reset via our [admin-requests repo](https://github.com/conda-forge/admin-requests?tab=readme-ov-file#reset-your-feedstock-token).
+- Request that any new packages be added to the allowed outputs for the feedstock
+  via our [admin-requests repo](https://github.com/conda-forge/admin-requests?tab=readme-ov-file#add-a-package-output-to-a-feedstock).
```

Comment:
This point is in the wrong section now.

Reply:
It's already in the previous section. Removed it from here